docs/development.adoc

```diff
@@ -354,14 +354,33 @@ $ make test-odo-login-e2e
 
 NOTE: `make test-odo-login-e2e` doesn't honour environment variable `TEST_EXEC_NODES`. So by default it runs login and logout command integration test suite on a single ginkgo test node sequentially to avoid race conditions in a parallel run.
 
-* For the entire integration test suite:
+Entire integration tests suite can be run in parallel and sequentially separately:
+
+* To run entire integration tests spec in parallel (default: 2 ginkgo test node), on a test cluster:
```

Comment:
Should we also mention how someone can set the value to something other than 2 if it's relevant for their environment?

Reply:
Sure, i can put a note or a place where it fits better in the doc

Reply:
Done